packages/itwinui-css/src/progress-indicator/linear.scss

```diff
@@ -12,7 +12,7 @@
     width: 100%;
     height: $height;
     overflow: hidden;
-    background-color: var(--iui-color-background-border);
+    background-color: var(--iui-color-background-5);
```

Comment:
Linear and Radial were using different color variables.  `var(--iui-color-background-border)` had too low contrast, switched to `var(--iui-color-background-5)` to align with radial and have better contrast.